backend/zeno_backend/server.py

```diff
@@ -308,18 +337,43 @@ def get_slice_table(req: SliceTableRequest, request: Request):
         if not util.project_access_valid(project_uuid, request):
             return Response(status_code=401)
 
-        filter_sql = table_filter(project_uuid, req.model, slice.filter_predicates)
+        filter_sql = table_filter(
+            project_uuid, slice_table_request.model, slice.filter_predicates
+        )
 
-        sql_table = select.slice_table(project_uuid, filter_sql, req)
+        sql_table = select.slice_table(project_uuid, filter_sql, slice_table_request)
```

Comment:
Nit: maybe make slice a param here as well? Might need a refactor anyway once we throw exceptions in select.